app/controllers/api/v1/graphql_controller.rb

```diff
@@ -21,6 +21,13 @@ def query
         )
         render :json => result
       end
+
+      private
+
+      # RBAC readonly access is allowed for graphql's POST
+      def request_is_readonly
+        true
```

Comment:
We'll need to remember to update this when we allow graphql to be used for create/update